modules/optimization/doc/content/modules/optimization/examples/shapeOpt_Annulus.md

```diff
@@ -0,0 +1,122 @@
+# Constrained Optimization Example: Shape Optimization
+
+## Background
+
+The MOOSE Optimization module provides a flexible framework for solving
+optimization problems in MOOSE. This page will demonstrate constrained shape
+optimization of an annulus while satisfying an equality volume constraint.
+
+# Example: Shape Optimization
```

Comment:
I would reorganize this section a little bit. Under the main header, state the algorithm and multiapp structure. Then I would go with the forward problem description first, then the optimization app.